src/Microsoft.PowerShell.Commands.Utility/resources/TestJsonCmdletStrings.resx

```diff
@@ -127,6 +127,6 @@
     <value>The JSON is not valid with the schema.</value>
   </data>
   <data name="JsonSchemaFileOpenFailure" xml:space="preserve">
-    <value>JSON schema file open failure</value>
+    <value>Can not open JSON schema file: {0}</value>
```

Comment:
Now you should pass the file name as parameter to get it here.

Reply:
Sure, sorry for the delay. Done.